configs/config-ccm-psk-tls1_2.h

```diff
@@ -73,13 +73,6 @@
         MBEDTLS_TLS_PSK_WITH_AES_256_CCM_8,             \
         MBEDTLS_TLS_PSK_WITH_AES_128_CCM_8
 
-/*
- * Save RAM at the expense of interoperability: do this only if you control
- * both ends of the connection!  (See comments in "mbedtls/ssl.h".)
- * The optimal size here depends on the typical size of records.
- */
-#define MBEDTLS_SSL_MAX_CONTENT_LEN             512
```

Comment:
Please revert this change. This is a major change affecting almost all existing users

Reply:
This and 6 of other comments are only saying "don't try to eliminate MBEDTLS_SSL_MAX_CONTENT_LEN"... I removed that part of the patches.  That leaves the allocator restricted to MBEDTLS_MAX_CONTENT_LEN... so changing it from 16384 will defeat the point of the new code computing it needs a bigger buffer, and continue to break if the peer sends something larger.  It might be a good idea to adapt the comment on that when the dust settles saying it is either deprecated or preferable to let the dynamic stuff discover the necessary buffer size.